CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Add a view to cancel reminder mails, link added in the footer of mails
 - Prevent multiple join to BBB meetings
 - Introduce anonymous_id mechanism
+- Persist live video participants
```

Comment:
In the commit message:

`s/are used to update the video object wich now caontains those lists./are used to update the video object wich now contains those lists./`